src/index.js

```diff
@@ -3,18 +3,11 @@ import 'dotenv/config'
 
 import { handlerReplyBtn, addButtonAction } from './components/handlers/buttonHandler.js'
 import { getLessonsInfo } from './components/handlers/scheduleHandler.js'
-import { commands } from './commandsList.js'
-import { commandHandler } from './commandHandler.js'
+import registerBotCommands from './commands/registerBotCommands.js'
 
 const bot = new Telegraf(process.env.TELEGRAM_TOKEN, { polling: true })
 
-//Command processing
-Object.keys(commandHandler).forEach(command => {
-	bot.command(command, commandHandler[command])
-})
-
-//Commands list
-bot.telegram.setMyCommands(commands)
+registerBotCommands(bot)
```

Comment:
@mkangoo Now related to commands logic is placed in `commands` directory. If we want to add something related to commands and separate it to different file it should be placed somewhere here (ofc if you accept the PR)

Advantages:
1. If you want to add new command you just add new entry to `commands` array in `registerBotCommand.js`
2. Code in `index.js` now look much  cleaner declarative